src/helpers/defaultConnectors.ts

```diff
@@ -10,7 +10,6 @@ import { WebWalletConnector } from "../connectors/webwallet"
 export const defaultConnectors = ({
   argentMobileOptions,
   webWalletUrl,
-  provider,
```

Comment:
can we still inject a custom provider if you remove this?

Reply:
We don't inject a provider anymore with the new SNIP. Providers are something the dapp devs use themselves from sn.js